jobs/build/promote/Jenkinsfile

```diff
@@ -683,6 +683,27 @@ node {
                     echo "Skipping PR creation for non-x86 CPU arch"
                 }
             }
+
+            stage("validate RHSAs") {
+                if (params.DRY_RUN) {
+                    return
+                }
+                if (advisory == -1) {
+                    return
+                }
+                if (major == 4 && !is_4stable_release) {
+                    return
+                }
+                release.getAdvisoryIds().each {
+                    res = commonlib.shell(
+                        script: "${buildlib.ELLIOTT_BIN} --group ${group} verify-rhsa ${it}",
```

Comment:
s/verify-rhsa/validate-rhsa/ ?

Also I don't think "--group ${group} " is strictly required.